cypress/e2e/bulk-edit/csv/bulk-edit-location.cy.js

```diff
@@ -0,0 +1,135 @@
+        });
+      });
+    });
+  });
+
+  after('Delete test data', () => {
+    cy.getAdminToken().then(() => {
+      ServicePoints.deleteViaApi(testData.servicePoint.id);
+      InventoryInstances.deleteInstanceAndHoldingRecordAndAllItemsViaApi(item.barcode);
+      Users.deleteViaApi(testData.user.userId);
+      FileManager.deleteFile(`cypress/fixtures/${itemBarcodesFileName}`);
+      FileManager.deleteFileFromDownloadsByMask(matchedRecordsFileName);
+      FileManager.deleteFileFromDownloadsByMask(previewFileName);
+      FileManager.deleteFileFromDownloadsByMask(changedRecordsFileName);
+      Locations.deleteViaApi(testData.defaultLocation);
+    });
+  });
+
+  it(
+    'C357064 Verify Items bulk edit if the location name contains "/" (firebird) (TaaS)',
+    { tags: [TestTypes.extendedPath, DevTeams.firebird] },
+    () => {
+      // Select "Inventory-items" record type => Select "Items barcode" from "Record identifier" dropdown
+      BulkEditSearchPane.checkItemsRadio();
+      BulkEditSearchPane.selectRecordIdentifier('Item barcode');
+      // Upload a .csv file with items barcodes (see Preconditions) by dragging it on the file drag and drop area
+      BulkEditSearchPane.uploadFile(itemBarcodesFileName);
+      BulkEditSearchPane.checkForUploading(itemBarcodesFileName);
+      BulkEditSearchPane.waitFileUploading();
+      // Check the result of uploading the .csv file with Items barcodes
+      BulkEditSearchPane.verifyMatchedResults(item.barcode);
+      // Click "Actions" menu => Check checkboxes (if not yet checked) next to:
+      BulkEditActions.openActions();
+      BulkEditSearchPane.changeShowColumnCheckboxIfNotYet(
+        'Item permanent location',
+        'Item temporary location',
+      );
+      BulkEditSearchPane.verifyResultColumTitles('Item permanent location');
+      BulkEditSearchPane.verifyResultColumTitles('Item temporary location');
+      // Click "Actions" menu => Select "Download matched records (CSV)" element
+      BulkEditActions.openActions();
+      BulkEditActions.downloadMatchedResults();
+      ExportFile.verifyFileIncludes(matchedRecordsFileName, [
+        item.barcode,
+        'Permanent Location',
+        'Temporary Location',
+      ]);
+      // Click "Actions" menu => Select "Start Bulk edit" option
+      BulkEditActions.openInAppStartBulkEditFrom();
+      // Select "Temporary item location" from "Select option" dropdown => Select "Replace with" option from  "Select action" dropdown
+      BulkEditActions.replaceTemporaryLocation(locationName);
+      // Click on the "Plus" icon
+      BulkEditActions.addNewBulkEditFilterString();
+      // Select "Permanent item location" from  "Select option" dropdown on the new row => Select "Replace with" option from  "Select action" dropdown
+      BulkEditActions.replacePermanentLocation(locationName, 'item', 1);
+      // Click "Confirm changes" button
+      BulkEditActions.confirmChanges();
```

Comment:
Review all step's expected results. Lack of assertions

Reply:
Done!